### ISSUE TITLE
A bunch of localized legal-docs point to English versions of things

### ISSUE THREAD

**Issue by pdehaan** [mozilla/legal-docs#185]
Similar to #153, but since I was grepping anyways, I thought I'd write some disgusting bash shell scripts to try and find all instances of "/en-us/" in non "en-US.md" files.

It's not pretty, but probably roughly accurate.

``` sh
$ grep -irn "/en-us/" --count . | grep -v "^./.git/" | grep -v ":0$" | grep -iv "en-US.md"
./firefox_cloud_services_ToS/cs.md:1
./firefox_cloud_services_ToS/de.md:1
./firefox_cloud_services_ToS/el.md:1
./firefox_cloud_services_ToS/es.md:1
./firefox_cloud_services_ToS/fr.md:1
./firefox_cloud_services_ToS/hi.md:1
./firefox_cloud_services_ToS/hr.md:1
./firefox_cloud_services_ToS/hu.md:1
./firefox_cloud_services_ToS/it.md:1
./firefox_cloud_services_ToS/ja.md:1
./firefox_cloud_services_ToS/mk.md:1
./firefox_cloud_services_ToS/pl.md:1
./firefox_cloud_services_ToS/pt-BR.md:1
./firefox_cloud_services_ToS/ru.md:1
./firefox_cloud_services_ToS/sr-LATN.md:1
./firefox_cloud_services_ToS/ta.md:1
./firefox_cloud_services_ToS/zh-CN.md:1
./firefox_os_privacy_notice/bn-BD.md:1
./firefox_os_privacy_notice/bn-IN.md:1
./firefox_os_privacy_notice/cs.md:1
./firefox_os_privacy_notice/fr.md:1
./firefox_os_privacy_notice/hi.md:1
./firefox_os_privacy_notice/hr.md:1
./firefox_os_privacy_notice/ja.md:1
./firefox_os_privacy_notice/mk.md:1
./firefox_os_privacy_notice/ru.md:1
./firefox_os_privacy_notice/ta.md:1
./heartbleed_privacy_notice_en.US.md:1
./marketplace_developer_agreement/bn-BD.md:5
./marketplace_developer_agreement/bn-IN.md:5
./marketplace_developer_agreement/cs.md:5
./marketplace_developer_agreement/el.md:5
./marketplace_developer_agreement/es.md:1
./marketplace_developer_agreement/fr.md:5
./marketplace_developer_agreement/hi.md:5
./marketplace_developer_agreement/hr.md:5
./marketplace_developer_agreement/hu.html:10
./marketplace_developer_agreement/hu.md:5
./marketplace_developer_agreement/ja.md:5
./marketplace_developer_agreement/mk.md:5
./marketplace_developer_agreement/pl.md:1
./marketplace_developer_agreement/pt-BR.md:6
./marketplace_developer_agreement/ru.md:5
./marketplace_developer_agreement/sr.md:5
./marketplace_developer_agreement/ta.md:5
./marketplace_developer_agreement/zh-CN.md:5
./marketplace_privacy_policy/bn-BD.md:2
./marketplace_privacy_policy/bn-IN.md:2
./marketplace_privacy_policy/cs.md:2
./marketplace_privacy_policy/de.md:1
./marketplace_privacy_policy/es.md:1
./marketplace_privacy_policy/fr.md:2
./marketplace_privacy_policy/hi.md:2
./marketplace_privacy_policy/hr.md:2
./marketplace_privacy_policy/hu.md:2
./marketplace_privacy_policy/it.md:1
./marketplace_privacy_policy/ja.md:2
./marketplace_privacy_policy/mk.md:2
./marketplace_privacy_policy/pl.md:2
./marketplace_privacy_policy/ru.md:2
./marketplace_privacy_policy/sr.md:1
./marketplace_privacy_policy/ta.md:2
./marketplace_privacy_policy/zh-CN.md:2
./marketplace_terms_of_use/bn-BD.md:2
./marketplace_terms_of_use/bn-IN.md:2
./marketplace_terms_of_use/cs.md:2
./marketplace_terms_of_use/fr.md:2
./marketplace_terms_of_use/hi.md:2
./marketplace_terms_of_use/hr.md:2
./marketplace_terms_of_use/hu.md:2
./marketplace_terms_of_use/it.md:2
./marketplace_terms_of_use/ja.md:2
./marketplace_terms_of_use/mk.md:2
./marketplace_terms_of_use/ru.md:2
./marketplace_terms_of_use/sr.md:3
./marketplace_terms_of_use/ta.md:2
./marketplace_terms_of_use/zh-CN.md:2
./Mobile_Partner_Website_Agreements/zh-CN.md:7
./mozilla_privacy_policy/bn-IN.md:1
./mozilla_privacy_policy/cs.md:1
./mozilla_privacy_policy/de.md:1
./mozilla_privacy_policy/es.md:1
./mozilla_privacy_policy/fr.md:1
./mozilla_privacy_policy/hi.md:1
./mozilla_privacy_policy/hr.md:1
./mozilla_privacy_policy/hu.md:1
./mozilla_privacy_policy/mk.md:1
./mozilla_privacy_policy/pl.md:1
./mozilla_privacy_policy/ru.md:1
./mozilla_privacy_policy/sr.md:1
./mozilla_privacy_policy/ta.md:1
./mozilla_privacy_policy/zh-CN.md:1
./README.md:5
```

---

**TL;DR:** Basically we have [/firefox_cloud_services_ToS/it.md](https://github.com/mozilla/legal-docs/blob/master/firefox_cloud_services_ToS/it.md) which has the following contents:

> _L'[informativa sulla privacy nella pagina Firefox Cloud Services](http://www.mozilla.org/en-US/privacy/firefox-cloud/) descrive..._

The Italian doc is pointing to an _/en-US/_ page instead of an Italian translation. It should probably be normalized to http://www.mozilla.org/privacy/firefox-cloud/ without the locale in the URL.

---

**UPDATE _(Oct 18, 2017)_:** Here's a list of recent files with "en-US" (minus files in "marketplace_*" and "firefox_os_*"):

```sh
$ git grep -i -l "en-US" | egrep -v (marketplace_|firefox_os_)

Firefox-Addons-Distribution-Agreement-20150522
Mobile_Partner_Website_Agreements/zh-CN.md
README.md
WebRTC_PrivacyNotice/ro.md
WebRTC_ToS/ro.md
firefox_cloud_services_ToS/ar.md
firefox_cloud_services_ToS/bn-BD.md
firefox_cloud_services_ToS/bn-IN.md
firefox_cloud_services_ToS/de.md
firefox_cloud_services_ToS/el.md
firefox_cloud_services_ToS/es-ES.md
firefox_cloud_services_ToS/fr.md
firefox_cloud_services_ToS/hi.md
firefox_cloud_services_ToS/hr.md
firefox_cloud_services_ToS/hu.md
firefox_cloud_services_ToS/it.md
firefox_cloud_services_ToS/ja.md
firefox_cloud_services_ToS/pl.md
firefox_cloud_services_ToS/pt-BR.md
firefox_cloud_services_ToS/ro.md
firefox_cloud_services_ToS/ru.md
firefox_cloud_services_ToS/sr.md
firefox_cloud_services_ToS/ta.md
firefox_cloud_services_ToS/zh-CN.md
klar_privacy_notice/en-DE.md
websites_privacy_notice/cs.md
websites_privacy_notice/hr.md
websites_privacy_notice/hu.md
websites_privacy_notice/sr.md
```

**Comment by MikkCZ** [comment]
This is the current list of localized documents containing `/en-us` (case insensitive). English versions should be covered #966 already.
```
Mobile_Partner_Website_Agreements/zh-CN.md
README.md
WebRTC_PrivacyNotice/ro.md
WebRTC_ToS/ro.md
firefox_cloud_services_ToS/ar.md
firefox_cloud_services_ToS/bn-BD.md
firefox_cloud_services_ToS/bn-IN.md
firefox_cloud_services_ToS/cs.md
firefox_cloud_services_ToS/de.md
firefox_cloud_services_ToS/el.md
firefox_cloud_services_ToS/es-ES.md
firefox_cloud_services_ToS/fr.md
firefox_cloud_services_ToS/hi.md
firefox_cloud_services_ToS/hr.md
firefox_cloud_services_ToS/hu.md
firefox_cloud_services_ToS/it.md
firefox_cloud_services_ToS/ja.md
firefox_cloud_services_ToS/pl.md
firefox_cloud_services_ToS/pt-BR.md
firefox_cloud_services_ToS/ro.md
firefox_cloud_services_ToS/ru.md
firefox_cloud_services_ToS/sr.md
firefox_cloud_services_ToS/ta.md
firefox_cloud_services_ToS/zh-CN.md
klar_privacy_notice/en-DE.md
marketplace_developer_agreement/ar.md
marketplace_developer_agreement/bn-BD.md
marketplace_developer_agreement/bn-IN.md
marketplace_developer_agreement/cs.md
marketplace_developer_agreement/de.md
marketplace_developer_agreement/el.md
marketplace_developer_agreement/en-US.md
marketplace_developer_agreement/es-ES.md
marketplace_developer_agreement/hi.md
marketplace_developer_agreement/hr.md
marketplace_developer_agreement/hu.md
marketplace_developer_agreement/it.md
marketplace_developer_agreement/mk.md
marketplace_developer_agreement/pl.md
marketplace_developer_agreement/pt-BR.md
marketplace_developer_agreement/ro.md
marketplace_developer_agreement/ru.md
marketplace_developer_agreement/sr.md
marketplace_developer_agreement/sw.md
marketplace_developer_agreement/ta.md
marketplace_developer_agreement/tl-PH.md
marketplace_privacy_policy/ar.md
marketplace_privacy_policy/ja.md
marketplace_privacy_policy/sw.md
marketplace_privacy_policy/tl-PH.md
marketplace_terms_of_use/af.md
marketplace_terms_of_use/ar.md
marketplace_terms_of_use/bn-BD.md
marketplace_terms_of_use/bn-IN.md
marketplace_terms_of_use/en-US.md
marketplace_terms_of_use/es-ES.md
marketplace_terms_of_use/fr.md
marketplace_terms_of_use/hi.md
marketplace_terms_of_use/it.md
marketplace_terms_of_use/ru.md
marketplace_terms_of_use/sw.md
marketplace_terms_of_use/ta.md
marketplace_terms_of_use/tl-PH.md
marketplace_terms_of_use/xh.md
marketplace_terms_of_use/zu.md
websites_privacy_notice/cs.md
websites_privacy_notice/hr.md
websites_privacy_notice/hu.md
websites_privacy_notice/sr.md
```
